.github/workflows/publish-release-image-wrapper.yaml

```diff
@@ -1,12 +1,32 @@
 name: "Create and Publish Release Docker Image"
 
 on:
-  workflow_call:
   workflow_dispatch:
-
 jobs:
-  to_be_filled_out:
-    steps:
-      - name: This workflow will be filled out in https://github.com/tenstorrent/tt-metal/pull/15013
-        run: |
-          echo "NOOP"
+  static-checks:
```

Comment:
`static-checks` don't need to be here, and `build-artifact` is done in a previous job. It's not functionally incorrect, but wasteful of compute

Reply:
Removing static checks. 
This is a wrapper workflow so there is no previous job for `build-artifact`.